lib/msf/core/payload/windows/reverse_http.rb

```diff
@@ -447,12 +447,61 @@ def asm_reverse_http(opts={})
       ^
     end
 
+
+    if defined?(dynamic_stage_size?) && dynamic_stage_size?
```

Comment:
The indentation looks off around this line? Also I think we talked about this being the future proof solution. If that's what you went with could we add a comment about what changes we're ready for in the future?

Reply:
Done